src/services/http.ts

```diff
@@ -1,10 +1,12 @@
-import { Logger } from './logger';
+import { Logger } from "./logger";
 
-import type { ApiConfig } from '../types';
+import type { ApiConfig } from "../types";
 export class HTTP {
   logger: Logger;
   apiConfig: ApiConfig;
 
+  static $inject = ["logger", "apiConfig"];
+
   constructor(apiConfig: ApiConfig) {
```

Comment:
you inject "logger" and "apiConfig" with ioc container, but don't use them like arguments